libcloud/test/common/test_openstack_identity.py

```diff
@@ -379,6 +379,12 @@ def test_get_user_without_email(self):
         self.assertEqual(user.name, 'userwithoutemail')
         self.assertEqual(user.email, None)
 
+    def test_get_user_without_enabled(self):
+        user = self.auth_instance.get_user(user_id='c')
+        self.assertEqual(user.id, 'c')
+        self.assertEqual(user.name, 'userwithoutenabled')
+        self.assertEqual(user.enabled, None)
```

Comment:
you could have done `self.assertIsNone` here

Reply:
@vdloo thank you for catching this!

I've globally switched this out for `assertIsNone` in https://github.com/apache/libcloud/pull/1264